Show enterprise installs for Cloud scope MacOS Installs (#19669)
gravitational/teleport: docs/pages/includes/install-homebrew.mdx

 <Notice type="danger">

The Teleport package in Homebrew is not maintained by Teleport and we can't
guarantee its reliability or security. We recommend the use of our [official
Teleport packages](https://goteleport.com/teleport/download?os=mac).

<ScopedBlock scope={["enterprise", "cloud"]}>

> The Homebrew version lacks the ability to maintain SAML/OIDC Authentication
> connector through the `tctl` binary. Administrators especially are recommended
> to install the official enterprise maintained versions.

Reviewer: ```suggestion
The Homebrew version of Teleport lacks the ability to maintain SAML/OIDC authentication
connectors through the `tctl` binary. We recommend installing the official Teleport Enterprise edition of `tctl`.
```
- Plural agreement and capitalization
- Since Teleport encourages admin-less RBAC and doesn't have a built-in admin role, I think we can ditch "Administrators" here and use the active voice with "We recommend".

>   <Admonition type="note" scope={["enterprise", "cloud"]} scopeOnly>

Reviewer 2: I think we should make the homebrew section much smaller since we don't officially support it, it's not released along with our other artifacts, and it doesn't support a number of features.

A small note that says "there is a community maintained formula but we recommend using the official releases" should be sufficient IMO.

Reviewer 3: Such a note exists below:

>  The Teleport package in Homebrew is not maintained by Teleport and we can't
>  guarantee its reliability or security. We recommend the use of our [official
>  Teleport packages](https://goteleport.com/teleport/download?os=mac).

I'll consolidate while editing
